tracer/src/Datadog.Trace/TraceContext.cs

```diff
@@ -140,6 +139,9 @@ public void CloseSpan(Span span)
 
             if (spansToWrite.Count > 0)
             {
+                // When receiving chunks of spans, the backend checks whether the aas.resource.id tag is present on any of the
+                // span to decide which metric to emit (datadog.apm.host.instance or datadog.apm.azure_resource_instance one).
+                AddAASMetadataToTraceChunk(spansToWrite.Array[0]);
```

Comment:
You'll need to update this usage since we renamed the method :) 
```suggestion
                AddAASMetadata(spansToWrite.Array[0]);
```